pypolychord/settings.py

```diff
@@ -120,6 +120,10 @@ class PolyChordSettings:
         (Default: exp(-1))
         How often to update the files and do clustering
 
+    synchronous : boolean
+        (Default: True)
+        Parallelise with synchronous workers.
+
```

Comment:
I think a bit more info for the user would be helpful. At least something on how this is very relevant in case speed varies across param space. Maybe also something on how this changes computation for someone unfamiliar with what "synchronous" means?

Reply:
OK, I've updated this now -- let me know if that is clearer.